web-local/src/lib/components/leaderboard/LeaderboardOptionsMenu.svelte

```diff
@@ -21,7 +21,12 @@
   bind:active={optionsMenuActive}
   let:toggleFloatingElement
 >
-  <IconButton on:click={toggleFloatingElement}>
+  <IconButton
+    on:click={(e) => {
```

Comment:
```suggestion
    on:click|stopPropagation={(e) => {
```

Think this should work as well

Reply:
ah, good call, let's merge this then.